types/prismjs/index.d.ts

```diff
@@ -142,10 +142,14 @@ export interface LanguageDefinition {
     rest?: Token[];
 }
 
-export interface Languages {
+export type Languages
```

Comment:
I got the following error with that definition.

```
TypeScript error: node_modules/@types/prismjs/index.d.ts(145,18): Error TS2411: Property 'extend' of type '(id: string, redef: LanguageDefinition) => LanguageDefinition' is not assignable to string index type 'LanguageDefinition'.
TypeScript error: node_modules/@types/prismjs/index.d.ts(145,18): Error TS2411: Property 'insertBefore' of type '(inside: string, before: string, insert: LanguageDefinition, root: LanguageDefinition) => Languag...' is not assignable to string index type 'LanguageDefinition'.
```

also see https://github.com/Microsoft/TypeScript/issues/17763